lib/internal/source_map/source_map.js

```diff
@@ -0,0 +1,289 @@
+// Copyright 2013 the V8 project authors. All rights reserved.
```

Comment:
👍 done, I'll probably also publish a copy to npm, once this is a bit more tested, so that it's easier for the community to iterate on (we can upstream any bug fixes to SourceMap.js).